papers/bradley_dice/55_bradley_dice.rst

```diff
@@ -0,0 +1,553 @@
+..   .. latex::
+..      :usepackage: footmisc
+
+:author: Bradley D. Dice
+:email: bdice@umich.edu
+:institution: Department of Physics, University of Michigan, Ann Arbor
+:institution: These authors have an equal contribution to the work presented.
+
+:author: Brandon L. Butler
+:email: butlerbr@umich.edu
+:institution: Department of Chemical Engineering, University of Michigan, Ann Arbor
+:institution: These authors have an equal contribution to the work presented.
+
+:author: Vyas Ramasubramani
+:email: vramasub@umich.edu
+:institution: Department of Chemical Engineering, University of Michigan, Ann Arbor
+
+:author: Alyssa Travitz
+:email: atravitz@umich.edu
+:institution: Macromolecular Science and Engineering Program, University of Michigan, Ann Arbor
+
+:author: Michael M. Henry
+:email: mikehenry@u.boisestate.edu
+:institution: Micron School of Materials Science and Engineering, Boise State University
+
+:author: Hardik Ojha
+:email: hojha@ch.iitr.ac.in
+:institution: Department of Chemical Engineering, Indian Institute of Technology Roorkee
+
+:author: Carl S. Adorf
+:email: TODO
+:institution: Department of Chemical Engineering, University of Michigan, Ann Arbor
+
+:author: Sharon C. Glotzer
+:email: sglotzer@umich.edu
+:institution: Department of Physics, University of Michigan, Ann Arbor
+:institution: Department of Chemical Engineering, University of Michigan, Ann Arbor
+:institution: Biointerfaces Institute, University of Michigan, Ann Arbor
+
+:bibliography: paper
+
+-------------------------------------------------------------------
+signac: Data Management and Workflows for Computational Researchers
+-------------------------------------------------------------------
+
+.. class:: abstract
+
+The signac data management framework (https://signac.io) helps researchers execute reproducible computational studies, scales workflows from laptops to supercomputers, and emphasizes portability and fast prototyping.
+With signac, users can track, search, and archive data and metadata for file-based workflows and automate workflow submission on high performance computing (HPC) clusters.
+We will discuss recent improvements to the software's feature set, scalability, scientific applications, usability, and community.
+Newly implemented synced data structures, features for generalized workflow execution, and performance optimizations will be covered, as well as recent research using the framework and the project's efforts on improving documentation, contributor onboarding, and governance.
```

Comment:
I don't necessarily like the wording but something like this needs to be mentioned I think since it does take up a half page and isn't covered elsewhere. Maybe something like
```suggestion
Newly implemented synced data structures, features for generalized workflow execution, and performance optimizations will be covered, as well as recent research using the framework and changes to the project's outreach and governance as a response to its growth.
```

Reply:
I agree with @b-butler here